src/styles/utils/_variables.scss

```
@@ -0,0 +1,13 @@
+$block-width: 100px;
+$block-height: 100px;
+$block-column: 7;
+$gap: 1px;
+
+$days: ("mon" 0,
+"tue" 1,
+"wed" 2,
+"thu" 3,
+"fri" 4,
+"sat" 5,
+"sun" 6
+);
```

tiny readability comment
```suggestion
$days: 
  "mon" 0,
  "tue" 1,
  "wed" 2,
  "thu" 3,
  "fri" 4,
  "sat" 5,
  "sun" 6;
```